ACMG Standards and guidelines for interpretation of sequence variants
========================================================
author: Jun Kang
date: 2019.09.25
autosize: true

Interpretation of variants
========================================================
* Pathogenic  
* Likely-pathogenic  
* Uncertain (VUS)  
* Likely-benign  
* Benign  

Standards and Guidelines for the Interpretation of Sequence Variants
========================================================
* To describe variants identified in **Mendelian disorders**  
* ENIGMA BRCA1/2 Gene Variant Classification Criteria
* **American College of Medical Genetics and Genomics (ACMG)**
* International Agency for Research on Cancer (IARC)

ACMG Standards and guidelines for interpretation of sequence variants
========================================================
![plot of chunk unnamed-chunk-1](https://media.nature.com/lw926/nature-assets/gim/journal/v17/n5/images/gim201530f1.jpeg)

Population, disease-specific, and sequence databases
=========================================================
![plot of chunk unnamed-chunk-2](https://media.nature.com/full/nature-assets/gim/journal/v17/n5/images/gim201530t1.jpeg)


In silico predictive algorithms
=========================================================
![plot of chunk unnamed-chunk-3](https://media.nature.com/full/nature-assets/gim/journal/v17/n5/images/gim201530t2.jpeg)

Criteria for classifying pathogenic variants
=========================================================
![plot of chunk unnamed-chunk-4](https://media.nature.com/full/nature-assets/gim/journal/v17/n5/images/gim201530t3.jpeg)

Criteria for classifying benign variants
=========================================================
![plot of chunk unnamed-chunk-5](https://media.nature.com/full/nature-assets/gim/journal/v17/n5/images/gim201530t3.jpeg)

Rules for combining criteria to classify sequence variants
=========================================================
![plot of chunk unnamed-chunk-6](https://media.nature.com/full/nature-assets/gim/journal/v17/n5/images/gim201530t4.jpeg)

Criteria for classifying pathogenic variants
=========================================================
![plot of chunk unnamed-chunk-7](https://media.nature.com/full/nature-assets/gim/journal/v17/n5/images/gim201530t5.jpeg)

Example
========================================================
* Temporal bone abnormalities detected by computed tomography in a hearing-impaired patient
* Uncertain variant in SLC26A4, the gene associated with Pendred syndrome
* Testing other family members to establish when a variant is **de novo**
* Variant **cosegregates** with disease in the family
* Variant is in **trans with a pathogenic variant** in the same recessive disease-causing gene
* Filtering out or discounting the vast majority of variants for dominant diseases when they can be observed in healthy relatives is possible

Making the interpretation much more efficient and conclusive

CMC pathology lab
=======================================================
* https://www.ncbi.nlm.nih.gov/clinvar/variation/51049/
* https://brcaexchange.org/variants?search=p.Arg3384Ter
* https://gnomad.broadinstitute.org/variant/13-32972800-C-T

KOBRA
=======================================================
![](KOBRA_head.png)
![](KOBRA.png)

End truncation
=======================================================
![](BRCA2.png)

HUMAN MUTATION Mutation in Brief 31: E 1200-E1240 (2010) Online
========================================================
Finally, three sequence variants – BRCA2 c.9976A>T (BIC: K3326X), c.10095delCins11 (BIC: 10323delCins11) and c.10150C>T (BIC: **R3384X**) predicted to result in protein truncation were ruled as exceptions that **could not be classified** because of their **location near the 3´-end** and possibly dispensable part of the gene.

Conclusions
========================================================
* Interpretaion of sequence variants
* Knowledge base (ClinVar, BRCAexchange ...)
* VUS  

