<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>High allele frequency of somatic mutation</title>
    <meta charset="utf-8" />
    <meta name="author" content="Jun Kang" />
    <link href="Allele_files/remark-css-0.0.1/default.css" rel="stylesheet" />
    <link href="Allele_files/remark-css-0.0.1/chocolate.css" rel="stylesheet" />
    <link href="Allele_files/remark-css-0.0.1/chocolate-fonts.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# High allele frequency of somatic mutation
### Jun Kang
### 2019 11 1

---






# Question 1

&gt; Can we determine whether the variant is germline or somatic using allele frequency and tumor percentage in tumor only test?

---
# BRCA result

* BRCA1 mutation: Positive - p.Glu649Ter (c.1945G&gt;T)  
* Variant allele frequency 63.84%  
* Tumor cell percentage: 70%  
* BRCA negative in blood sample

---
# Question 2

&gt; Is it possible **64% allele frequency** of somatic mutation?

---
class: center
# Variant allele frequency in clinical tumor sample

`$$Allele \space frequency \approx Read \space count$$`
![:scale 70%](img/Allele_3.png)  

---
class: center
# Somatic variant, Two copy, Tumor cellularity 50%

![](img/Allele_4.png)  
Variant allele frequency?

---
class: center
# Germline variant, Heterozigosity, Two copy, Tumor cellularity 25%

![](img/Allele_2.png)  
Variant allele frequency?

class: center
# Germline variant, Heterozigosity, Two copy, Tumor cellularity 50%

![](img/Allele_1.png)  
Variant allele frequency?

---
# Hypothesis

Germline, homo | Germline, hetero | Somatic
:-----: | :----: | :----------------------:
 100% |  50% |  half of tumor cellularity, `\(\le 50\%\)`

---
class: center
# Germline variant, Heterozigosity, One copy, LOH, Tumor cellularity 50%

![](img/Allele_5.png)  
Variant allele frequency?

---
class: center
# Germline variant, Heterozigosity, One copy, LOH, Tumor cellularity 25%

![](img/Allele_6.png)  
Variant allele frequency?

---
class: center
# Somatic variant, One copy, LOH, Tumor cellularity 50%

![](img/Allele_8.png)  
Variant allele frequency?

---
class: center
# Somatic variant, Amplification, Tumor cellularity 25%

![:scale 70%](img/Allele_9.png)  
Variant allele frequency?


---
# Variant allele frequency in clinical tumor sample

* Germline vs somatic  
* Tumor cell proportion  
* Loss of heterozygosity 
* Copy number  

---
# Allele frequency in Somatic vs Germline in tumor only sample

* BRCA1 mutation: Positive - p.Glu649Ter (c.1945G&gt;T)  
* Variant allele frequency 63.84%  
* Tumor cell percentage: 70%  

---
# Allele frequency[1]

`\(AF_{germline}\)` `\(=\)` `\({pV+1-p} \over {pC+2(1-p)}\)`  
`\(AF_{somatic}\)` `\(=\)` `\(pV \over pC+2(1-p)\)` 

* Given copy number (C)
* Variant allele count (V)
* Sample purity (p)
* Variant status (somatic or germline)

---
# Limited information

What we know | What we don't know
:-------------:|:-----------------:
Tumor cell percentage | Copy number
Variant allele frequency  | LOH

---
# Error

* Tumor cell percentage
* Allele frequency  

`$$Allele \space frequency \not\approx Read \space count$$`

---
# BRCA mutation and LOH [2]

![:scale 80%](LOH.png)

---
# Allele frequency 

* BRCA1 mutation: Positive - p.Glu649Ter (c.1945G&gt;T)  
* Variant allele frequency 63.84%  
* Tumor cell percentage: 70% -&gt; 80%  
* Somatic (supposed)  
* LOH (supposed)  
* One copy (supposed)  

---
class: center
# Allele frequency 

![:scale 70%](img/Allele_12.png)

`\(AF_{somatic}\)` `\(=\)` `\(pV \over pC+2(1-p)\)` `\(= 0.67\)`

         
---
# Allele frequency 

* BRCA1 mutation: Positive - p.Glu649Ter (c.1945G&gt;T)  
* Variant allele frequency 63.84%  
* Tumor cell percentage: 70% -&gt; 60%  
* Germline (supposed)  
* LOH (supposed)  
* One copy (supposed)  

---
class: center
# Allele frequency 

![:scale 60%](img/Allele_13.png)

`\(AF_{germline}\)` `\(=\)` `\({pV+1-p} \over {pC+2(1-p)}\)` `\(= 0.71\)`


---
# Someone ask ...

&gt;Does tumor only NGS test distinguish a variant somatic and germline?  
Answer 'No'  

&gt;Is it possible **64% allele frequency** of somatic BRCA variant?  
Answer 'Yes, It is when BRCA somatic variant accompanied with LOH'  

---
class: center
# Germline variant, One copy, LOH, Tumor cellularity 100% (Bonus1)

Allele frequency 100% **hetero germline** variant  

![:scale 60%](img/Allele_7.png)  

---
class: center
# Somatic variant, Amplification, Hetero, Tumor cellularity 100% (Bonus2)

Allele frequency 90% **somatic** variant  

![:scale 70%](img/Allele_11.png)  

---
class: my-one-page-font
# References

[1] J. X. Sun, Y. He, E. Sanford, et al. "A computational approach to distinguish somatic vs. germline origin of genomic alterations
from deep sequencing of cancer specimens without a matched normal". En. In: _PLOS Computational Biology_ 14.2 (2018), p. e1005965.
ISSN: 1553-7358.

[2] M. M. de Jonge, D. Ruano, R. van Eijk, et al. "Validation and Implementation of BRCA1/2 Variant Screening in Ovarian Tumor
Tissue". In: _The Journal of Molecular Diagnostics_ 20.5 (9. 2018), pp. 600-611. ISSN: 1525-1578.

&lt;style type="text/css"&gt;
.remark-slide-content {
    font-size: 32px;
    padding: 1em 4em 1em 4em;
}

.my-one-page-font {
  font-size: 18px;
}

.beginners{
visibility: hidden;
}

.intermediate{
visibility: hidden;
}

.advanced{
visibility: visible;
}
 {
    max-width: 100%;
}
&lt;/style&gt;
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="macro.js"></script>
<script>var slideshow = remark.create();
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
